etl/steps/data/garden/wb/2022/wb_gender/wb_gender.countries.json

```diff
@@ -0,0 +1,225 @@
+  "Channel Islands": "Channel Islands",
+  "Chile": "Chile",
+  "China": "China",
+  "Colombia": "Colombia",
+  "Comoros": "Comoros",
+  "Congo, Dem. Rep.": "Democratic Republic of Congo",
+  "Congo, Rep.": "Congo",
+  "Costa Rica": "Costa Rica",
+  "Cote d'Ivoire": "Cote d'Ivoire",
+  "Croatia": "Croatia",
+  "Cuba": "Cuba",
+  "Curacao": "Curacao",
+  "Cyprus": "Cyprus",
+  "Czech Republic": "Czechia",
+  "Denmark": "Denmark",
+  "Djibouti": "Djibouti",
+  "Dominica": "Dominica",
+  "Dominican Republic": "Dominican Republic",
+  "Ecuador": "Ecuador",
+  "Egypt, Arab Rep.": "Egypt",
+  "El Salvador": "El Salvador",
+  "Equatorial Guinea": "Equatorial Guinea",
+  "Eritrea": "Eritrea",
+  "Estonia": "Estonia",
+  "Eswatini": "Eswatini",
+  "Ethiopia": "Ethiopia",
+  "Faroe Islands": "Faeroe Islands",
+  "Fiji": "Fiji",
+  "Finland": "Finland",
+  "France": "France",
+  "French Polynesia": "French Polynesia",
+  "Gabon": "Gabon",
+  "Gambia, The": "Gambia",
+  "Georgia": "Georgia",
+  "Germany": "Germany",
+  "Ghana": "Ghana",
+  "Gibraltar": "Gibraltar",
+  "Greece": "Greece",
+  "Greenland": "Greenland",
+  "Grenada": "Grenada",
+  "Guam": "Guam",
+  "Guatemala": "Guatemala",
+  "Guinea": "Guinea",
+  "Guinea-Bissau": "Guinea-Bissau",
+  "Guyana": "Guyana",
+  "Haiti": "Haiti",
+  "Honduras": "Honduras",
+  "Hong Kong SAR, China": "Hong Kong",
+  "Hungary": "Hungary",
+  "Iceland": "Iceland",
+  "India": "India",
+  "Indonesia": "Indonesia",
+  "Iran, Islamic Rep.": "Iran",
+  "Iraq": "Iraq",
+  "Ireland": "Ireland",
+  "Isle of Man": "Isle of Man",
+  "Israel": "Israel",
+  "Italy": "Italy",
+  "Jamaica": "Jamaica",
+  "Japan": "Japan",
+  "Jordan": "Jordan",
+  "Kazakhstan": "Kazakhstan",
+  "Kenya": "Kenya",
+  "Kiribati": "Kiribati",
+  "Korea, Dem. People's Rep.": "North Korea",
+  "Korea, Rep.": "South Korea",
+  "Kosovo": "Kosovo",
+  "Kuwait": "Kuwait",
+  "Kyrgyz Republic": "Kyrgyzstan",
+  "Lao PDR": "Laos",
+  "Latvia": "Latvia",
+  "Lebanon": "Lebanon",
+  "Lesotho": "Lesotho",
+  "Liberia": "Liberia",
+  "Libya": "Libya",
+  "Liechtenstein": "Liechtenstein",
+  "Lithuania": "Lithuania",
+  "Luxembourg": "Luxembourg",
+  "Macao SAR, China": "Macao",
+  "Madagascar": "Madagascar",
+  "Malawi": "Malawi",
+  "Malaysia": "Malaysia",
+  "Maldives": "Maldives",
+  "Mali": "Mali",
+  "Malta": "Malta",
+  "Marshall Islands": "Marshall Islands",
+  "Mauritania": "Mauritania",
+  "Mauritius": "Mauritius",
+  "Mexico": "Mexico",
+  "Micronesia, Fed. Sts.": "Micronesia (country)",
+  "Moldova": "Moldova",
+  "Monaco": "Monaco",
+  "Mongolia": "Mongolia",
+  "Montenegro": "Montenegro",
+  "Morocco": "Morocco",
+  "Mozambique": "Mozambique",
+  "Myanmar": "Myanmar",
+  "Namibia": "Namibia",
+  "Nauru": "Nauru",
+  "Nepal": "Nepal",
+  "Netherlands": "Netherlands",
+  "New Caledonia": "New Caledonia",
+  "New Zealand": "New Zealand",
+  "Nicaragua": "Nicaragua",
+  "Niger": "Niger",
+  "Nigeria": "Nigeria",
+  "North America": "North America",
```

Comment:
World Bank's definition of North America is different from OWID's definition. I suppose the mapping should be `"North America": "North America (WB)"`. The same may apply to other regions (although not to income groups).